runtime/common/src/evm/precompile.rs

```diff
@@ -95,6 +95,10 @@ where
 	}
 }
 
+/// Altair's precompiles
+/// For now, Altair uses the exact same set of precompiles used in Development.
+pub type Altair<R> = Development<R>;
```

Comment:
Given that they meant to be the exact same set for now, I made this an alias; shall we need a different set for Altair, we just need to change this file alone in the future.